webdriver/tests/find_element_from_shadow_root/user_prompts.py

```diff
@@ -0,0 +1,129 @@
+# META: timeout=long
+
+import pytest
+
+from tests.support.asserts import (
+    assert_error,
+    assert_same_element,
+    assert_success,
+    assert_dialog_handled,
+)
+
+
+def find_element(session, shadow_id, using, value):
+    return session.transport.send(
+        "POST", "session/{session_id}/shadow/{shadow_id}/element".format(
+            session_id=session.session_id,
+            element_id=shadow_id),
+        {"using": using, "value": value})
+
+
+@pytest.fixture
+def check_user_prompt_closed_without_exception(session, create_dialog, get_shadow_page):
```

Comment:
Please move the fixtures to the conftest.py file

Reply:
Done in 9e70d8f.